File: translator/test_files/test_cases/localProperty.js
foo = {};
(function(foo){
  foo.y = 3;
  foo.f = function(a){
    var x = 42;
    var y = 50;
    return y;
  }
  ;
  foo.box = function(){
    return foo.f(foo.y);
  }
  ;
}
(foo));

File: js/js.translator/testData/box/expression/misc/localProperty.kt
// EXPECTED_REACHABLE_NODES: 489
package foo

val y = 3

fun f(a: Int): Int {
    val x = 42
    val y = 50

    return y
}

fun box(): String {
    val r = f(y)
    return if (r == 50) "OK" else "Fail, r = $r"
}
